.github/workflows/ci-backend.yml

```diff
@@ -264,21 +263,17 @@ jobs:
                   group: ${{ matrix.group }}
                   person-on-events: ${{ matrix.person-on-events }}
 
-            - uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
-              id: add-and-commit
-              if: ${{ matrix.clickhouse-server-image == 'clickhouse/clickhouse-server:22.3' && !matrix.person-on-events }}
+            - uses: EndBug/add-and-commit@v9
+              # Skip on forks
+              # Also skip in extra variant runs (FOSS or persons-on-events), as we want to ignore snapshots diverging there
+              if: ${{ !matrix.person-on-events && !matrix.foss && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Why not on FOSS? 🤔

Reply:
Foss and ee tests should now have separate snapshots. @EDsCODE have a look here too, I think things should be sorted after the latest change, and we'd want both?

Reply:
Nice, this errored before but seems to be supported now